korean/ask-kin/ask-add-100.md

```diff
@@ -0,0 +1,11 @@
+여태까지 초딩들에게 답변을 해 주면서 모은 내공을 탏탏 털어넣었다...
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.